aresponses/main.py

```diff
@@ -59,15 +64,15 @@ def __init__(self, *, scheme=sentinel, host="127.0.0.1", **kwargs):
     async def _handler(self, request):
         return await self._find_response(request)
 
-    def add(self, host, path=ANY, method=ANY, response="", match_querystring=False):
+    def add(self, host, path=ANY, method=ANY, response="", match_querystring=False, body=NoBody()):
```

Comment:
Seems confusing to have `body` at the end of the arguments but it would also be confusing anywhere.   Not sure how to address this. Perhaps a totally different signature.  

I'm starting to wonder if we should not implement this feature.

Reply:
same feeling for the signature, putting at the back just for backward compatibility

it's fine if the feature is not suitable, I have internal registry to use it already, let me know if you have some decision so I can address it different way or just leave it here for someone else who needs it to patch